Apps/CommonData/test/unit/CommonDataTests.csproj

```diff
@@ -15,6 +15,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="DeepEqual" Version="4.2.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="7.0.0" />
```

Comment:
Can we see if we can add the abstractions package instead?

Reply:
@ytqsl That won't work.  I need ConfigurationBuilder, which is part of Microsoft.Extensions.Configuration and not Microsoft.Extensions.Configuration.Abstractions